.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,90 @@
+name: Release Documentation
+
+on:
+  workflow_call:
+    inputs:
+      tag_name:
+        description: "Release Tag to use"
+        type: "string"
+        required: true
+        default: "debug"
+      dry_run:
+        description: "Flag for testing"
+        type: "boolean"
+        default: false
+        required: false
+
+env:
+  PAGE_DIR: "./tmp/page"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release-docs:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Checkout - page branch
+        uses: actions/checkout@v3
+        with:
+          ref: "page"
+          path: ${{ env.PAGE_DIR }}
+
+      - name: Get Latest Release Information
+        uses: octokit/request-action@v2.x
+        id: latest_release
+        with:
+          route: GET /repos/:owner/:repository/releases/latest
+          repository: lifecycle-toolkit
+          owner: keptn
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Detect needed Folder Operations
+        id: folder
+        run: |
+            CURRENT_VERSION=`cat ${{ env.PAGE_DIR }}/docs/content/en/docs/version || echo "unknown"`
+            echo $CURRENT_VERSION
+            if [[ "${{ fromJson(steps.latest_release.outputs.data).tag_name }}" == "${{ inputs.tag_name }}" ]]; then
+              if [[ "${{ fromJson(steps.latest_release.outputs.data).tag_name }}" != "$CURRENT_VERSION" ]]; then
+                echo "migrate=docs-$CURRENT_VERSION" >> $GITHUB_OUTPUT
+              fi
+              echo "name=docs" >> $GITHUB_OUTPUT
+              echo "latest=true" >> $GITHUB_OUTPUT
+            else
+              echo "name=docs-${{ inputs.tag_name }}" >> $GITHUB_OUTPUT
+            fi
+
+      - name: Move old latest release documentation folder
+        if: ${{ steps.folder.outputs.migrate }}
+        run: |
+          FROM="${{ env.PAGE_DIR }}/docs/content/en/docs"
+          TO="${{ env.PAGE_DIR }}/docs/content/en/${{ steps.folder.outputs.migrate }}"
+          echo "Moving ${FROM} -> ${TO}"
+          cp -R $FROM $TO
+          rm -rf $FROM
+
+      - name: Copy configuration
+        if: ${{ steps.folder.outputs.latest }}
+        run: |
+          cp ./docs/* "${{ env.PAGE_DIR }}/docs" || true
```

Comment:
why `|| true`?

Reply:
if `cp` finds directories and it has no `-r` flag, it returns 1 - I will add a comment

Reply:
but why not add `-r` then? it will find folders for sure

Reply:
but then we would copy the real docs twice 😓 
this is to copy any update of the README, package.json etc

Reply:
You can see the outcome with `-r` here: https://github.com/keptn/lifecycle-toolkit/actions/runs/6769506602/job/18396032122?pr=2420
It creates two documentations :(

Reply:
`cp` without flags cannot copy directories, either we need to use `cp -a` or `cp -r`

Reply:
we don't need `-a` since we don't need to preserve attributes and owners